Rakefile

```diff
@@ -1,11 +1,8 @@
-require_relative 'app/environment'
+require_relative "app/environment"
+require "rspec/core/rake_task"
```

Comment:
`rspec` (`govuk_test`) is listed as a development dependency which is why this needs to be part of the `begin`/`rescue`/`end` block.

We're seeing errors in integration:

```
/data/apps/cache-clearing-service/shared/bundle/ruby/2.6.0/bin/rake:23:in `<main>'
/data/vhost/cache-clearing-service/releases/20200310134449/Rakefile:2:in `require': cannot load such file -- rspec/core/rake_task (LoadError)
from /data/vhost/cache-clearing-service/releases/20200310134449/Rakefile:2:in `<top (required)>'
from /data/vhost/cache-clearing-service/shared/bundle/ruby/2.6.0/gems/rake-13.0.1/lib/rake/rake_module.rb:29:in `load'
```

Reply:
Thanks for notifying. Raised: https://github.com/alphagov/cache-clearing-service/pull/260